cbmc/proofs/IotMqtt_Connect/README.md

```diff
@@ -0,0 +1,141 @@
+# Memory safety proof for IotMqtt_Connect
+
+This proof harness attains 92% code coverage.  The following comments explain
+why the uncovered lines of code are unreachable code.
+
+Some functions contain unreachable blocks of code:
+
+* libraries/standard/mqtt/src/iot_mqtt_helper.c
```

Comment:
I found that including the file name made the whole thing harder to read.  I now like the style of just listing the function and counting on the coverage list in the report to give the file names.    Also run this through pandoc to make sure it builds as you expect. I think the sublists must be indented 4 spaces and not 2.

Reply:
Done.